lib/MusicBrainz/Server/Data/Artist.pm

```diff
@@ -398,23 +398,13 @@ __PACKAGE__->meta->make_immutable;
 no Moose;
 1;
 
-=head1 COPYRIGHT
+=head1 COPYRIGHT AND LICENSE
 
 Copyright (C) 2009 Lukas Lalinsky
-Copyright (C) 2011 MetaBrainz Foundation
+Copyright (C) 2019 MetaBrainz Foundation
```

Comment:
Thanks for fixing these to use the shorter version. :) I would generally leave the publication dates untouched unless the file is rewritten, though (or if substantial modifications are made, just append the current date).

Reply:
Oh, I was wondering about that. Mostly I've seen the date just been changed (but not sure *when* to change it) and some have 2013-15 or whatever (whatever *that* means)...